package.json

```diff
@@ -18,19 +20,23 @@
     "eslint-plugin-standard": "3.0.1",
     "istanbul": "0.4.5",
     "mocha": "2.5.3",
-    "supertest": "1.1.0"
+    "supertest": "1.1.0",
+    "typescript": "2.8.1"
   },
   "files": [
     "LICENSE",
     "HISTORY.md",
-    "index.js"
+    "index.js",
+    "index.d.ts"
   ],
+  "typings": "index.d.ts",
```

Comment:
You can remove this as mentioned above, since it is now the standard name :+1:

Reply:
Cool, though you needed to make explicit either way.

Reply:
No you don't. Please remove it.

Reply:
Sure, will remove it!